src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -14479,9 +14505,6 @@ Map {
         "isRequired": true,
         "type": "func",
       },
-      "onHighligtedIndexChange": Object {
-        "type": "func",
-      },
```

Comment:
Was previously used when we copied ComboBox over to CarbonComboBox. Refactored ComboBox no longer needs this override. CarbonComboBox was removed, we now use the stock component from Carbon.